### ISSUE TITLE
libsass build failure - fails to check for libtool

### ISSUE THREAD

**Issue by hmoffatt** [microsoft/vcpkg#17657]
**Host Environment**
- OS: Linux (Debian buster)
- Compiler: g++ 8.3

**To Reproduce**
Without system libtool installed:
`./vcpkg install libsass`

**Failure logs** 
```
[autoconf-x64-linux-err.log](https://github.com/microsoft/vcpkg/files/6420528/autoconf-x64-linux-err.log)
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:105 (message):
    Command failed: /usr/bin/autoreconf -vfi
    Working Directory: /home/hamish/tmp/vcpkg/vcpkg-2021.04.30/buildtrees/libsass/src/3f75a82396-feba490666.clean/
    Error code: 1
    See logs for more information:
      /home/hamish/tmp/vcpkg/vcpkg-2021.04.30/buildtrees/libsass/autoconf-x64-linux-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_make.cmake:600 (vcpkg_execute_required_process)
  ports/libsass/portfile.cmake:10 (vcpkg_configure_make)
  scripts/ports.cmake:142 (include)
```

**Additional context**
Problem is solved by installing libtool through the system package manager, but the port fails to check for this.

**Comment by hmoffatt** [comment]
Also requires but fails to check for automake (for the aclocal) command - discovered this on OSX.

**Comment by JackBoosY** [comment]
Please provide failure log _ /home/hamish/tmp/vcpkg/vcpkg-2021.04.30/buildtrees/libsass/autoconf-x64-linux-err.log_.

Thanks.

**Comment by hmoffatt** [comment]
Sorry I thought I had attached it to the original message.
[autoconf-x64-linux-err.log](https://github.com/microsoft/vcpkg/files/6430892/autoconf-x64-linux-err.log)

**Comment by JackBoosY** [comment]
```
src/GNUmakefile.am:27: error: Libtool library used but 'LIBTOOL' is undefined
src/GNUmakefile.am:27:   The usual way to define 'LIBTOOL' is to add 'LT_INIT'
src/GNUmakefile.am:27:   to 'configure.ac' and run 'aclocal' and 'autoconf' again.
src/GNUmakefile.am:27:   If 'LT_INIT' is in 'configure.ac', make sure
src/GNUmakefile.am:27:   its definition is in aclocal's search path.
```

Can you please try install libtool first? `sudo apt-get install libtool`.

**Comment by hmoffatt** [comment]
Yes it works if libtool is installed. If autoconf is missing you get a nice error, but not if libtool is missing.

The same problem occurs for automake (if aclocal is missing) on osx.

**Comment by JackBoosY** [comment]
cc @Neumann-A, we should look for autoconf and libtool in advance in osx I think.

**Comment by hmoffatt** [comment]
Same problem on Linux (Debian) - that is where I saw the libtool issue.

**Comment by AYakymov1990** [comment]
error: building libsass:arm64-osx failed with: BUILD_FAILED
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+libsass
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?title=[libsass]+Build+error&body=Copy+issue+body+from+%2FUsers%2Foleksandrpc%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md

**Comment by jleaders** [comment]
I can confirm this is still an issue on Ubuntu LTS 22.04, and is resolved by `sudo apt install libtool -y` (autoconf is needed as well)
